src/Data/Vector/Generic/Sized.hs

```diff
@@ -324,9 +326,9 @@ instance KnownNat n => Applicative (Vector Boxed.Vector n) where
 -- @'join' :: Vector n (Vector n a) -> Vector n a@ gets the /diagonal/ from
 -- a square "matrix".
 instance KnownNat n => Monad (Vector Boxed.Vector n) where
-  return   = replicate
```

Comment:
May as well remove these instead. Save some trouble whenever we get the monad of no return proposal in Haskell.